app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,48 @@
+class PostsController < ApplicationController
+  def index
+    # @current_user = current_user
+    @user = User.find(params[:user_id])
+    @post = Post.where(author_id: params[:user_id])
+    # @post = @user.posts
+
+    # @post = Post.includes(:comments: [:comments, { comments: [:author] }]).find(params[:user_id])
+
+    # @user = User.includes(:posts, posts: [:comments,
+    #  { comments: [:author] }]).find(params[:user_id])
+    # @posts = Post.where(author_id: params[:user_id]).includes(:user)
+    render '/posts/index'
+  end
+
```

Comment:
- [ ] Kindly remove all commented codes in this file. Removing commented codes makes your work look clean and readable. 👍 🖥️